apps/cowswap-frontend/src/pages/AdvancedOrders/index.tsx

```diff
@@ -24,8 +24,14 @@ import {
 } from 'modules/twap'
 import { TwapFormState } from 'modules/twap/pure/PrimaryActionButton/getTwapFormState'
 
+import { SHOW_LIMIT_ORDERS_PROMO } from 'common/constants/featureFlags'
+import { limitOrdersPromoDismissedAtom } from 'common/state/limitOrdersPromoAtom'
+
 export default function AdvancedOrdersPage() {
-  const { isUnlocked } = useAtomValue(advancedOrdersAtom)
+  const { isUnlocked: isWidgetUnlocked } = useAtomValue(advancedOrdersAtom)
+  const isDismissed = useAtomValue(limitOrdersPromoDismissedAtom)
+  const shouldShowPromo = SHOW_LIMIT_ORDERS_PROMO && !isDismissed
```

Comment:
This is TWAP module but it uses const that belongs to limit orders, that's not right